BredaUniversityGames/Y2024-25-PR-BB
# Feature/directional light component

### Description

- Directional light is now a component in the ECS
- Added color functionality to the light
- Removed SceneDescription from GPUScene as it is no longer needed there

### Issues

[Task in Codecks](https://bubonic-brotherhood.codecks.io/milestones/run/40/card/15a-directional-light-component)

### Test criteria

1. Open the program
2. Inside the world inspector there should be an entity with the `Directional Light` name
3. Select it and play around with the values
4. The directional light should change accordingly
